en/guide/extending_csd.md

```diff
@@ -67,25 +74,29 @@ int CameraDeviceCustom::init(CameraParameters &camParam)
 }
 ```
 
-**Action** : The `CameraDeviceCustom` must handle setting of the parameters declared and resetting of all the parameters.
+**Action**: The `CameraDeviceCustom` must handle setting and resetting of the declared parameters.
```

Comment:
I presume setting is done in init() above(?). What function do they do the resetting in?